scripts/deploy-contracts-live.sh

```diff
@@ -38,8 +52,12 @@ _store_code() {
   local store_result
   store_result=$(run_cmd "$nolus_home_dir" tx wasm store "$wasm_code_path" --instantiate-anyof-addresses "$instantiate_only_address" --from "$store_code_privileged_wallet_key" $FLAGS --yes --output json)
   store_result=$(echo "$store_result" | sed -n '/{/{p; q}')
-  local -r code_id=$(echo "$store_result" | jq -r '.logs[].events[] | select(.type == "store_code") | .attributes[] | select(.key == "code_id") | .value')
+  local -r store_tx_hash=$(echo "$store_result" | jq -r '.txhash')
+
+  _wait_tx_included_in_block "$nolus_home_dir" "$nolus_net" "$store_tx_hash"
 
+  local -r store_tx=$(run_cmd "$nolus_home_dir" q tx "$store_tx_hash" --node "$nolus_net" --output json)
```

Comment:
The code for obtaining the transaction is the same as when waiting for it at the `_wait_tx_included_in_block()`. Suggest extending the function to wait/block and return the tx once it appears in a block.  Then one may use the returned tx or abandon it.